credential-exchange-types/src/format.rs

```diff
@@ -260,3 +279,73 @@ pub enum FieldType {
     #[serde(untagged)]
     Unknown(String),
 }
+
+fn serialize_bool<S>(value: &bool, serializer: S) -> Result<S::Ok, S::Error>
+where
+    S: serde::Serializer,
+{
+    serializer.serialize_str(if *value { "true" } else { "false" })
+}
+
+fn deserialize_bool<'de, D>(deserializer: D) -> Result<bool, D::Error>
+where
+    D: serde::Deserializer<'de>,
+{
+    let value = String::deserialize(deserializer)?;
+    match value.as_str() {
+        "true" => Ok(true),
+        "false" => Ok(false),
+        _ => Err(serde::de::Error::custom("expected 'true' or 'false'")),
```

Comment:
Not sure how we want to handle EditableFields where the data is malformed. Skip the whole field?